integration-tests/src/router.rs

```diff
@@ -4370,197 +4370,6 @@ mod set_route {
 			});
 		}
 
-		#[test]
-		fn set_route_should_work_when_stored_route_is_broken_due_to_frozen_asset() {
```

Comment:
With the new change, if the assets are frozen, we still allow setting route for that. I wonder if this comes with some security implication, but probably not really as they are frozen anyway so no trade is allowed.

Reply:
worst case different route has to be set